packages/@uppy/companion/src/standalone/helper.js

```diff
@@ -81,7 +81,7 @@ const getConfigFromEnv = () => {
         useAccelerateEndpoint:
           process.env.COMPANION_AWS_USE_ACCELERATE_ENDPOINT === 'true',
         expires: parseInt(process.env.COMPANION_AWS_EXPIRES || '300', 10),
-        acl: process.env.COMPANION_AWS_DISABLE_ACL === 'true' ? null : (process.env.COMPANION_AWS_ACL || 'public-read'), // todo default to no ACL in next major and remove COMPANION_AWS_DISABLE_ACL
+        acl: process.env.COMPANION_AWS_ACL,
```

Comment:
Do we keep a document with all the breaking changes we are landing? Alternatively we could split all the breaking changes in different commits/PRs so it will appear on the CHANGELOGs.